RoslynPluginGenerator/Interfaces/INuGetPackageHandler.cs

```diff
@@ -28,6 +28,12 @@ public interface INuGetPackageHandler
         IPackage FetchPackage(string packageId, SemanticVersion version);
 
         /// <summary>
+        /// Returns the closure of packages required by the specified package
+        /// that have been installed locally
+        /// </summary>
+        IEnumerable<IPackage> GetInstalledDependencies(IPackage package);
```

Comment:
Added a new method to the interface to return all dependencies. Previously this was embedded in the analyzer generator. I've separated it out, partly because it makes testing simpler, and partly because we will need the list of packages that require licenses so we can display them to the user.